ekwipunek.py

```diff
@@ -0,0 +1,823 @@
+    """
+    def wyswietl(self):
+        return self
+
+class Ekwipunek:
+    """
+    Klasa Ekwipunek -> To w niej znajdują się Przedmioty z Podłogi dodane za pomocą
+    Bohatera. Tutaj również znajduje się kontener w uzyciu (nie wiem, czy słusznie).
+    :param brone: kontener na przedmioty klasy Broń
+    :param pancerze: kontener na przedmioty klasy Pancerz
+    :param przedmioty magiczne: kontener na przedmioty klasy Magia
+    :param pisma: kontener na przedmioty klasy Pismo
+    :param artefakty: kontener na przedmioty klasy Artefakt
+    :param jedzenie: kontener na przedmioty klasy Jedzenie
+    :param pozostale: kontener na przedmioty klasy Pozostale
+    :param magazyn: słownik zawierający wszystkie kontenery przypisane
+        do odpowiedniego klucza
+    :param w_uzyciu: kontener zawierający przedmioty znajdujące się w danej
+        chwili w użyciu
+    """
+    def __init__(self):
+        self._bronie = Bronie()
+        self._pancerze = Pancerze()
+        self._przedmioty_magiczne = PrzedmiotyMagiczne()
+        self._pisma = Pisma()
+        self._artefakty = Artefakty()
+        self._jedzenie = Zywnosc()
+        self._pozostale = Pozostalosci()
+        self._magazyn = {
+            "Bronie": self._bronie,
+            "Pancerze": self._pancerze,
+            "Przemioty Magiczne": self._przedmioty_magiczne,
+            "Pisma": self._pisma,
+            "Artefakty": self._artefakty,
+            "Jedzenie": self._jedzenie,
+            "Pozostałe": self._pozostale,
+        }
+        self._w_uzyciu = {}
+
+    @property
+    def bronie(self):
+        return self._bronie
+
+    @property
+    def pancerze(self):
+        return self._pancerze
+
+    @property
+    def przedmioty_magiczne(self):
+        return self._przedmioty_magiczne
+
+    @property
+    def pisma(self):
+        return self._pisma
+
+    @property
+    def artefakty(self):
+        return self._artefakty
+
+    @property
+    def jedzenie(self):
+        return self._jedzenie
+
+    @property
+    def pozostale(self):
+        return self._pozostale
+
+    @property
+    def w_uzyciu(self):
+        return self._w_uzyciu
+
+    @property
+    def magazyn(self):
+        return self._magazyn
+
+    def wyrzuc(self):
+        """
+        Metoda wyrzuć wyrzucająca dany przedmiot z obiektu klasy Ekwipunek i zwracająca go
+        do obiektu klasy Podłoga.
+        """
+        print("Przedmioty, które możesz wyrzucić, lub wpisz 'wyjscie', by wyjść:")
+        for key, value in self.wyswietl_magazyn().items():
+            for k, v in value.items():
+                print(k + " sztuk " + str(len(v)))
+        val = input("Podaj nazwę przedmiotu, który chcesz wyrzucić")
+        if val == "wyjscie":
+            pass
+        else:
+            for key, value in self.wyswietl_magazyn().items():
+                if val in value:
+                    przedmiot = self.magazyn[key][val][0]
+                    Podloga_Obiekt.zawartosc.append(przedmiot)
```

Comment:
To teraz dam Ci dopiero zadanie: 
Ta funkcja, jak każda inna nie powinna miec referencji do Podloga_Obiekt. Zatem powinna ona po prostu wyrzucac przedmiot ( czyli go zwracać ) bez wiedzy gdzie go wyrzuca. Teoretycznie w świecie gry, klasa Ekwipunek zawsze należy do klasy Świat, który ma swój "Ekwipunek" a raczej coś w rodzaju kontenera z lokalizacją. A więc można to zrobić na dwa sposoby ( które mi przychodzą do głowy ):
1. Wszelkie operacje typu wyrzuc, podnieś etc, powinny miec dodatkowy wrapper w postaci metody w klasie Swiat, które definiuje dostepne czynności: wyrzuc podnieś, naostrz, usiądź etc. A sam Ekwipunek i Podloga powinny być przypisane jako jego attrybuty
2. Klasa Ekwipunek wyrzuca jakiś przedmiot, dodaje go na kolejkę jak bufor, który go przyjmuje, a na drugim wątku ( Thread ) Obiekt Swiat monitoruje tą samą kolejkę i ściąga przedmioty z bufora i dodaje do swojego magazynu. Trzeba pamiętać w takim wypadku by zarządzać dostępem do kolejki, tj nie może być tak, że w tym samym czasie Świat coś ściąga a Ekwipunek coś dodaje, bo dostaniemy kolizję i prawdopodobnie OverflowError
Wybierz jedno :D

Reply:
A czym jest Podłoga jeśli nie kontenerem z lokalizacją? :D Tutaj chciałbym, żebyś rozwinął jak będziesz mieć moment

Reply:
### KLASA PODLOGA ZOSTAJE ZMIENIONA NA KLASE SWIAT, PO KTOREJ BEDA DZIEDZICZYC ODPOWIEDNIO KHORINIS I GORNICZA DOLINA. BOHATER/EKWIPUNEK (?) BEDZIE BRAC REFERENCJE DO TYCH OBIEKTOW I W TYCH OBIEKTACH BEDZIEMY OPEROWAC. TO ZNACZY ZE DANY PRZEDMIOT WYRZUCAMY DO DANEGO SWIATA

Reply:
W KAZDYM SWIECIE JEST JAKIES POLE PO KTORYM BOHATER MOZE SIE PORUSZAC -> PRZEDMIOTY MOGA BYC W JEGO ZASIEGU I TAK DALEJ... DO PRZEMYSLENIA